src/main/java/org/opentripplanner/standalone/config/RoutingRequestMapper.java

```diff
@@ -69,8 +70,14 @@ public static RoutingRequest mapRoutingRequest(NodeAdapter c) {
     request.boardSlack = c.asInt("boardSlack", dft.boardSlack);
     request.boardSlackForMode =
       c.asEnumMap("boardSlackForMode", TransitMode.class, NodeAdapter::asInt);
-    request.maxAccessEgressDurationSecondsForMode =
-      c.asEnumMap("maxAccessEgressDurationSecondsForMode", StreetMode.class, NodeAdapter::asDouble);
+    request.maxAccessEgressDurationForMode =
+      c.exist("maxAccessEgressDurationForMode")
+        ? c.asEnumMap("maxAccessEgressDurationForMode", StreetMode.class, NodeAdapter::asDuration)
+        : c.asEnumMap( // TODO: Remove deprecated parameter
```

Comment:
I think we should either log/throw an error or completely remove the backwards-compatibility.